doc/mgr/ansible.rst

```diff
@@ -96,80 +96,9 @@ And use the "active" manager node: ( "ceph -s" command in one monitor give you t
 Operations
 ==========
 
-**Inventory:**
+The available operations are:
 
-Get the list of storage devices installed in all the cluster nodes. The output format is::
+* device ls
+* create osd
+* remove osd
```

Comment:
Shouldn't that be "osd add" and "osd rm" ? You may want to spell those out entirely too, a'la:

`ceph orchestrator osd create ...`

Reply:
:-) ... yep... i like your proposal ... +1

> We agreed to use create/remove (See https://docs.google.com/document/d/17DIs_NCMBsTG3jbrhsG0Ot5y40TC9ADDZgEhmtF6Yks/edit#heading=h.wt65mijf6vf6)


if nobody against i will change it.

Reply:
Doesn't make sense to call it `remove`. It's called `rm` across all mon commands. See `ceph pool rm`, etc.

Reply:
"remove" has been replaced by "rm". I forget to change that in the ansible doc :-(, but the change is done in the command definition and in the orchestrator-cli doc. I will fix it

What about "s/create/add"?

Reply:
Create and add may have two distinct meanings. Let's revisit create vs add later on.